src/abstractdataframe/join.jl

```diff
@@ -212,8 +212,34 @@ function update_row_maps!(left_table::AbstractDataFrame,
     return to_bimap(left_ixs), to_bimap(leftonly_ixs), to_bimap(right_ixs), to_bimap(rightonly_ixs)
 end
 
+function check_uniqueness_func(check_uniqueness, joiner)
```

Comment:
Given that it's only called from one place, there's no point in defining  a separate function.

Reply:
I know I don't have to, but there is a point: good decomposition / style! It's a lot of clutter to leave in the core function definition, no?

Reply:
I think with my suggestion you should be able to make it shorter.